src/main/java/pl/marcin/project/WebAppRunner.java

```diff
@@ -0,0 +1,8 @@
+package pl.marcin.project;
+
+public class WebAppRunner implements AppRunner {
+    @Override
+    public void runApplication() {
+        System.out.println("Web is running...");
```

Comment:
Good start!
Here you can expand it using:
```
ConfigurableApplicationContext context = SpringApplication.run(ProjectApplication.class, args);
var service = context.getBean(??)  
```
and continue with that :)

Reply:
ok